Avatars :з implementation on Staff

jalonso12 opened the pull request:
- Avatars feature added!

Commit:
Avatars EP

- Created endpoints to load avatars from local folder

Commit:
Merge branch 'master' into ja_serve_avatars

Commit:
Avatars (Staff)

- Replaced "profile_pic" to "avatar" on staff table

- Changed "create employee" stored procedure so it could generate random number (1>=, <=10) on avatar attribute.

- Modified "create", "get all", "get specific" and added "update avtar" endpoints to support new table

Commit:
Merge branch 'ja_serve_avatars' of https://github.com/Gabuardi/the-spot-api into ja_serve_avatars

Gabuardi commented on src/routes/staff.js:
```diff
@@ -185,4 +185,40 @@ ROUTER.put('/role/:username', (request, response) => {
+  let sqlRequest = new sql.Request();
+
+  sqlRequest.input('id', employeeId);
+  sqlRequest.input('new_avatar', data.new_avatar);
```
Please change the name of the property `new_avatar` we should follow camelCase convention for JSON properties

Gabuardi commented on src/app.js:
```diff
@@ -57,11 +58,13 @@ APP.use('/books', booksRouter);
 APP.use('/songs', songsRouter);
 APP.use('/logs', logsRouter);
 APP.use('/customers', customersRouter);
+APP.use('/avatar', avatarsRouter);
+
+APP.use(express.static('resources'));
```
There is a better way to do the same which doesn't require a new module file
```suggestion
APP.use('/avatar',  express.static('resources/avatars'));
```
This is going to deliver the SVG files statically since the `avatars` directory, so we don't need a GET endpoint

Gabuardi reviewed (changes requested):
There is a couple of changes we need to do here, first of all

- [x] Rename PR to something that targets all the staff avatars functionality since now there is a single PR for both tasks.
- [x] Remove `__MACOSX` files into the avatars directory 
- [x] Update static serve for files
- [x] Fix JSON property of staff module

Commit:
Took out the trash

- Removed unnecessary files inside resources folder.

- Simplified avatar route

- Fixed endpoints camelCase

Commit:
Took out fellovers

- Removed avatars module
